py-polars/polars/lazyframe/frame.py

```diff
@@ -1830,7 +1867,7 @@ def sink_parquet(
         slice_pushdown: bool = True,
     ) -> DataFrame:
         """
-        Persists a LazyFrame at the provided path.
+        Collect and write a LazyFrame in streaming mode to a Parquet file at the path.
```

Comment:
I don't like the wording "collect to a file". We are explicitly not doing a `collect`, so this can be confusing.

Reply:
Let me know if you want me to make any changes

Reply:
How about "        Evaluate the query in streaming mode and write to a Parquet file at the path."